From bcb4b5355e73274722a6169dfa4f532353b3d6a7 Mon Sep 17 00:00:00 2001
From: XScreenSaver owners <xscreensaver-owner@fedoraproject.org>
Date: Wed, 9 Dec 2020 22:35:55 +0900
Subject: [PATCH] test-password.c: add skel definition for
 clientmessage_response

Fix below

/usr/bin/ld: /tmp/test-passwd.BnNU2t.ltrans0.ltrans.o: in function `passwd_event_loop':
/foo/xscreensaver-5.45/x86_64-pc-linux-gnu/driver/../../driver/lock.c:1937: undefined reference to `clientmessage_response'
collect2: error: ld returned 1 exit status
make: *** [Makefile:842: test-passwd] Error 1
---
 driver/test-passwd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/driver/test-passwd.c b/driver/test-passwd.c
index 9b4f98e..1c4dccc 100644
--- a/driver/test-passwd.c
+++ b/driver/test-passwd.c
@@ -67,6 +67,7 @@ void describe_monitor_layout (saver_info *si) { }
 Bool update_screen_layout (saver_info *si) { return 0; }
 Bool in_signal_handler_p = 0;
 char *timestring (time_t when) { return ""; }
+void clientmessage_response (saver_info *si, Window w, Bool error, const char *stderr_msg, const char *protocol_msg) {}
 
 const char *blurb(void) { return progname; }
 Atom XA_SCREENSAVER, XA_DEMO, XA_PREFS;
-- 
2.29.2

